src/viam/app/data_client.py

```diff
@@ -317,9 +317,9 @@ async def get_latest_tabular_data(
 
         Args:
             part_id (str): The ID of the part that owns the data.
-            resource_name (str): The name of the requested resource that captured the data.
-            resource_subtype (str): The subtype of the requested resource that captured the data.
-            method_name (str): The data capture method name.
+            resource_name (str): The name of the requested resource that captured the data. Ex: "my-sensor".
+            resource_subtype (str): The subtype of the requested resource that captured the data. Ex: "rdk:component:sensor".
+            method_name (str): The data capture method name. Ex: "Readings".
```

Comment:
Is there a data capture method called `Readings`? I'm not familiar with this method.

Reply:
Yes there is! I chose to include this as the example since this is the <component type, capture method> pair that has the most use for this endpoint in prod

<img width="351" alt="Screenshot 2024-12-17 at 3 02 14 PM" src="https://github.com/user-attachments/assets/c37d3b95-c556-4535-9684-8c5effb32ada" />
<img width="420" alt="Screenshot 2024-12-17 at 3 03 28 PM" src="https://github.com/user-attachments/assets/b6e2c6fe-e392-487e-aa01-894d926ccffc" />